cloudferry_devlab/cloudferry_devlab/tests/testcases/test_vm_migration.py

```diff
@@ -204,11 +208,18 @@ def compare_vm_parameter(param, vm1, vm2):
             for dst_vm in dst_vms:
                 if src_vm.vm_hash != dst_vm.vm_hash:
                     continue
-                compare_vm_parameter(param, src_vm, dst_vm)
+                if param == 'flavor':
+                    fail_msg.extend(self.compare_vm_flavor_parameter(
+                        'id', src_vm, dst_vm))
+                else:
+                    fail_msg.extend(compare_vm_parameter(
+                        param, src_vm, dst_vm))
```

Comment:
I think it would be better to write a separate test for flavors than complicate logic of existing